src/main/kotlin/calculator/domain/Operand.kt

```diff
@@ -0,0 +1,17 @@
+package calculator.domain
+
+@JvmInline
+value class Operand(val value: Int) {
+
+    constructor(value: String) : this(
+        try {
+            value.toInt()
+        } catch (e: NumberFormatException) {
+            throw RuntimeException("숫자가 아닌 값[$value] 입니다.")
+        }
+    )
+
+    init {
+        if (value < 0) throw RuntimeException("음수[$value]를 사용할 수 없습니다.")
```

Comment:
코틀린 감수성을 조금 추가해보자면 다음과 같이 변경해볼수 있습니다~!
```suggestion
        require(value < 0) { "음수[$value]를 사용할 수 없습니다." }
```